R/noms_date_range_pull.R

```diff
@@ -0,0 +1,33 @@
+library(googleAnalyticsR)
+library(googleAuthR)
+library(arrow)
+library(tidyr)
+library(lubridate)
+library(dplyr)
+
+yesterday <- Sys.Date()-1
+
+yesterday <- seq(from=yesterday, to=yesterday, by='days')
+
+source('R/pull_GA_data.R')
```

Comment:
hmm i guess maybe i don't want to do this. it seems to be doing a bunch of other data pulls I won't use